babel.config.json

```diff
@@ -1,4 +1,17 @@
 {
-  "presets": ["@babel/preset-react", "@babel/preset-env"],
-  "plugins": ["@babel/plugin-proposal-class-properties"]
+  "presets": [
+    "@babel/preset-react",
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "10"
```

Comment:
Ca sert à quoi le target node=10 ?

Reply:
Ca permet de faire des async / await

Reply:
https://medium.com/@binyamin/enabling-async-await-and-generator-functions-in-babel-node-and-express-71e941b183e2

Reply:
Si ça te va je te laisse fermer ce com

Reply:
Ce n'est pas plutôt ce plugin qu'il faut rajouter ? https://babeljs.io/docs/en/babel-plugin-transform-async-to-generator
Tom medium date de 2018 et comme on est sur babel 7 je pense que cela a changé.

Reply:
Yes j'avais mal cherché !! Merci @arnaudforaison

Reply:
Nan ben nan, ça marche pas (c'est pour le test en async / await)

Reply:
Ok c'est bon j'ai trouvé

https://github.com/facebook/jest/pull/7595#issuecomment-454486373